src/eureka/S5_lightcurve_fitting/models/BatmanModels.py

```diff
@@ -12,6 +12,62 @@
 from ...lib.split_channels import split
 
 
+class PlanetParams():
+    """
+    Define planet parameters.
+    """
+    def __init__(self, model, pid):
```

Comment:
No, we're passing in the `pid` each time `PlanetParams()` is called.  I've changed to utility of `PlanetParams()` to actually initialize the parameters to their correct values within the function, rather than being a dummy object that contains no real information.

Reply:
I've added documentation here and applied similar changes to `PoetModels.py`, `LorentzianModel.py`, and `DampedOscillatorModel.py`.